README.md

```diff
@@ -495,26 +504,26 @@ is compared as greatest. `false` to undo.
 #### legacy
 
 Set to `true` to force the package to be legacy instead of outdated.
-`false` to undo. Useful when a specific repository purposely contains
-an outdated version of specific project for compatibility purposes.
+Set to `false` to undo. Useful when a specific repository purposely contains
+an outdated version of a specific project for compatibility purposes.
 
 ```yaml
 - { name: ruby-slack-notifier-1, ruleset: aur, legacy: true }
 ```
 
 #### nolegacy
 
-Set to `true` to prevent the package to ever have legacy status.
+Set to `true` to prevent the package from ever having legacy status.
 This is useful for marking packages which declare to be of development
-version, but are never the less outdated.
+version, but are nevertheless outdated.
 
 ```yaml
 - { name: ffmpeg-git, nolegacy: true }
 ```
 
 #### warning
 
-Output a given warning when matched. Useful to catch places which
+Output a given warning when matched.
```

Comment:
@AMDmi3, here I removed an incomplete sentence, but if you provide me with the missing context, I'd be happy to complete it instead.